packages/db/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "../../tsconfig.json",
+    "compilerOptions": {
+        //"preserveSymlinks": true,
```

Comment:
why?

Reply:
three tsconfig.json files are in our vvplus and we need to extends them to each other.
reference:https://github.com/t3-oss/create-t3-turbo/blob/main/apps/nextjs/tsconfig.json